mautic/mautic
# TPROD 331 - GrapesJsBuilder: migrate views from PHP Engine to Twig

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
- Replaced the html.php templates with html.twig version
- Replaced references to the php templates with the new twig versions
- Did NOT remove references to slideshow.html.php `(MauticPageBundle:Page:Slots/slideshow.html.php)`. I believe this is okay, as another ticket which has been recently finished and merged (eg. PROD-322 PageBundle) also still references that php template. We can remove/replace these references when the component responsible is tackled (possibly CoreBundle? not sure)



#### Steps to test this PR:
In short: check that the GrapesJS builder still works and produces the expected content and html

- Open up a Mautic instance in the browser
- Make sure you have at least 1 contact in Mautic Contact
- In Channels -> Emails add a new Segment Email
- Select a theme made with some nice html or mjml (eg. Paprika)
- Open the Builder to see that everything in your template renders nicely
- Make some additions (eg. add a 2 col with a Text block and an image block)
- Save
- Open it again to see if it all still renders
- Close & send an email to one of your contacts
- Check that the mail is delivered as an html mail
 

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.
